.circleci/config.yml

```diff
@@ -118,19 +118,21 @@ jobs:
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
-      
+      - run:
+          name: pre-warm simulator
+          command:  |
+              open -a /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app --args -CurrentDeviceUDID "iPhone 8"
+              open -a /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app --args -CurrentDeviceUDID "iPhone 12"
+              open -a /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app --args -CurrentDeviceUDID "Apple TV"
       - install-dependencies
```

Comment:
`xcrun instruments` was removed from xcode 13. so instead we just run the simulator app manually.